gaffer-as-a-service/gaas-jwt-sidecar/src/test/java/uk/gov/gchq/gaffer/sidecar/controller/SidecarControllerTest.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.gchq.gaffer.sidecar.controller;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.reactive.WebFluxTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.reactive.server.WebTestClient;
+import org.springframework.web.reactive.function.BodyInserters;
+import uk.gov.gchq.gaffer.sidecar.util.UnitTest;
+
+@WebFluxTest(controllers = SidecarController.class)
+@UnitTest
+public class SidecarControllerTest {
+    @Autowired
+    private WebTestClient webClient;
+
+    @Test
+    void authEndpointReturns200WhenValidCredentials(){
```

Comment:
Tests should follow the format `shouldX` and have `// Given`, `// When`, `// Then`.

Reply:
Refactored